tests/robustness/linearizability_test.go

```diff
@@ -149,50 +69,53 @@ func TestRobustness(t *testing.T) {
 			clusterOfSize3Options = append(clusterOfSize3Options, e2e.WithSnapshotCatchUpEntries(100))
 		}
 		scenarios = append(scenarios, scenario{
-			name:      "ClusterOfSize3/" + traffic.name,
+			name:      "ClusterOfSize3/" + traffic.Name,
 			failpoint: RandomFailpoint,
 			traffic:   &traffic,
-			config:    *e2e.NewConfig(clusterOfSize3Options...),
+			cluster:   *e2e.NewConfig(clusterOfSize3Options...),
 		})
 	}
 	scenarios = append(scenarios, scenario{
 		name:      "Issue14370",
 		failpoint: RaftBeforeSavePanic,
-		config: *e2e.NewConfig(
+		cluster: *e2e.NewConfig(
 			e2e.WithClusterSize(1),
 			e2e.WithGoFailEnabled(true),
 		),
 	})
 	scenarios = append(scenarios, scenario{
 		name:      "Issue14685",
 		failpoint: DefragBeforeCopyPanic,
-		config: *e2e.NewConfig(
+		cluster: *e2e.NewConfig(
 			e2e.WithClusterSize(1),
 			e2e.WithGoFailEnabled(true),
 		),
 	})
 	scenarios = append(scenarios, scenario{
 		name:      "Issue13766",
 		failpoint: KillFailpoint,
-		traffic:   &HighTraffic,
-		config: *e2e.NewConfig(
+		traffic:   &traffic.HighTraffic,
+		cluster: *e2e.NewConfig(
 			e2e.WithSnapshotCount(100),
 		),
 	})
 	scenarios = append(scenarios, scenario{
 		name:      "Issue15220",
 		failpoint: RandomFailpoint,
-		traffic:   &ReqProgTraffic,
-		config: *e2e.NewConfig(
+		watch: watchConfig{
+			requestProgress: true,
+		},
+		traffic: &traffic.HighTraffic,
```

Comment:
Could you resolve all unrelated changes in a separate PR?  I expect this PR only focuses on moving all traffic related implementation into a dedicated package.

Reply:
Done